elections/tools/README.md

```diff
@@ -0,0 +1,37 @@
+
+# Election Tools
+
+This directory contains the tools required to generate the kata electorate
+
+## generate_electorate.py
+
+This tool need the following python libraries pytz github3.py pyyaml.  To
+install them in a virtutal envoronment do something like:
+
+```
+python3 -m venv .venv
```

Comment:
We normally prefix our code commands with `$ ` - have a peek at https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md which I think has some documentation style notes.